coralproject/talk
# [CORL-560] Update Admin > Config area to match new designs

## What does this PR do?

Updates styles for the Admin > Config UI layout.

## How do I test this PR?

- Head to the moderation area and see the fancy new styles.
- Compare them against Sam's designs that have been posted up on Drive
